WWDCFamily/Source/Map/Controllers/MapViewController.swift

```diff
@@ -1,18 +1,18 @@
 import UIKit
 import MapKit
 
-final class MapViewController: UIViewController, StoryboardInstance, RootChildViewController {
+final class MapViewController: UIViewController, RootChildViewController {
 
-    static let storyboardName: String = "Core"
+    private let datasource = MapDataSource()
 
-    @IBOutlet private weak var mapView: MKMapView! {
-        didSet {
-            mapView.setRegion(datasource.sfRegion, animated: true)
-            mapView.showsUserLocation = true
-        }
+    override func loadView() {
+        let view = UIView.instanceFromNib() as MapView
```

Comment:
The best part of having it like this is the separation of concerns and allowing dependency injection for view controllers with leads to better testability.

Controllers shouldn't be in charge if handling user input, views should be in charge of that. If you hook up a view controller to Interface Builder you lose dependency injection, meaning you can't do things like this:

```swift
let profileViewController = ProfileViewController(user: amanda)
```

Which is clean and testable.